README.md

```diff
@@ -659,6 +659,22 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+  * <a id='standard-library-type-shorthand'></a>(<a href='#standard-library-type-sugar'>link</a>) **For standard library types with a canonical shorthand form (`Optional`, `Array`, `Dictionary`), prefer using the shorthand form over the full generic form. [![SwiftFormat: typeSugar](https://img.shields.io/badge/SwiftFormat-typeSugar-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#typeSugar)
+
+  <details>
+
+  ```swift
+  // WRONG
+  let optional: Optional<String> = nil
+  let array: Array<String> = []
+  let dictionary: Dictionary<String, Any> = [:]
+
+  // RIGHT
+  let optional: String? = nil
+  let array: [String] = []
+  let dictionary: [String: Any] = [:]
```

Comment:
These examples violate [this](https://github.com/airbnb/swift#use-implicit-types) rule. Maybe we can use a different example?

Reply:
Not sure what you mean. I don't think we have an explicit preference between

```
let optional: String? = nil
let array: [String] = []
let dictionary: [String: Any] = [:]
```

and 

```
let optional = String?.none
let array = [String]()
let dictionary = [String: Any]()
```

(if that's what you're getting at -- not sure if you have something else in mind)

Reply:
I'm not sure I agree. These types cannot be inferred from the right hand side. I'd agree if the examples were using `let array: [String] = Array<String>()` but as is they need the explicit types.

Reply:
Yes, I was mistaken.